Cargo.lock

```diff
@@ -483,6 +483,7 @@ dependencies = [
  "if_chain",
  "itertools 0.9.0",
  "lazy_static 1.4.0",
+ "matches",
```

Comment:
I don't know if this was added automatically, on purpose, or is a merge issue, but I will verify and remove if needed.

(This PR is still a WIP and not ready for merge.)